src/WinRT.Runtime/Marshalers.cs

```diff
@@ -1255,6 +1255,23 @@ public static IObjectReference CreateMarshaler(object o, Guid delegateIID, bool
 
             return ComWrappersSupport.CreateCCWForObject<global::WinRT.Interop.IDelegateVftbl>(o, delegateIID);
         }
+
+        public static T FromAbi<T>(IntPtr nativeDelegate, Func<IntPtr, T> createCallback)

```

Comment:
Evaluating whether to keep callback or call create object here.

Reply:
Replaced callback with CreateRCW function that gets called by CreateObject.  Benchmark shows that in the existing delegate scenario that it brings better perf compared to the callback because we don't need to create an RCW each time even though there is an existing one.  And for the new delegate scenario, there is minimal differences (~50 ns).